test/core.controller.tests.js

```diff
@@ -1,16 +1,7 @@
 describe('Chart.Controller', function() {
 
 	function waitForResize(chart, callback) {
-		var resizer = chart.chart.canvas.parentNode._chartjs.resizer;
-		var content = resizer.contentWindow || resizer;
-		var state = content.document.readyState || 'complete';
-		var handler = function() {
-			Chart.helpers.removeEvent(content, 'load', handler);
-			Chart.helpers.removeEvent(content, 'resize', handler);
-			setTimeout(callback, 50);
-		};
-
-		Chart.helpers.addEvent(content, state !== 'complete'? 'load' : 'resize', handler);
+		setTimeout(callback, 50);
```

Comment:
Are there any other options besides the timeout for this case? @simonbrunel thoughts?